### PR TITLE
[docs] Updates pip installation command

### PR DESCRIPTION
**Pull request by captn3m0**
See https://github.com/deepmind/kapitan/issues/188 for more details

### CODE REVIEW

**Comment by ramaro** [comment]
Hey @captn3m0, I'm confused of what value `--only-binary` provides here?

**Comment by captn3m0** [comment]
It forces the wheel version to be installed, instead of the source version. The source version fails installation because of the missing `requirements.txt` file.

**Comment by captn3m0** [comment]
See this log for a sample installation in a fresh virtualenv that breaks:

```
mkvirtualenv ktest
Using base prefix '/usr'
New python executable in /home/nemo/.virtualenvs/ktest/bin/python
Installing setuptools, pip, wheel...
done.
virtualenvwrapper.user_scripts creating /home/nemo/.virtualenvs/ktest/bin/predeactivate
virtualenvwrapper.user_scripts creating /home/nemo/.virtualenvs/ktest/bin/postdeactivate
virtualenvwrapper.user_scripts creating /home/nemo/.virtualenvs/ktest/bin/preactivate
virtualenvwrapper.user_scripts creating /home/nemo/.virtualenvs/ktest/bin/postactivate
virtualenvwrapper.user_scripts creating /home/nemo/.virtualenvs/ktest/bin/get_env_details

pip install kapitan
Collecting kapitan
  Using cached https://files.pythonhosted.org/packages/ca/df/6e4c4e9dc1c905aa9a3297276297565a5c5044c441012949235979913913/kapitan-0.23.0.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-mfcvso8j/kapitan/setup.py", line 55, in <module>
        pkgs, new_links = install_deps()
      File "/tmp/pip-install-mfcvso8j/kapitan/setup.py", line 41, in install_deps
        with open('requirements.txt', 'r') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-mfcvso8j/kapitan/


pip install --no-cache kapitan

Collecting kapitan
  Downloading https://files.pythonhosted.org/packages/ca/df/6e4c4e9dc1c905aa9a3297276297565a5c5044c441012949235979913913/kapitan-0.23.0.tar.gz (91kB)
     |████████████████████████████████| 92kB 199kB/s 
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-t448nq__/kapitan/setup.py", line 55, in <module>
        pkgs, new_links = install_deps()
      File "/tmp/pip-install-t448nq__/kapitan/setup.py", line 41, in install_deps
        with open('requirements.txt', 'r') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-t448nq__/kapitan/
```

File: README.md [hunk 1]
```diff
@@ -64,13 +64,13 @@ Kapitan needs Python 3.6.
 User (`$HOME/.local/lib/python3.6/bin` on Linux or `$HOME/Library/Python/3.6/bin` on macOS):
 
 ```shell
-pip3 install --user --upgrade kapitan
```
Comment (adrianchifor) [inline README.md]: We still need `--user`

**Comment by adrianchifor** [comment]
What version of python are you using?

**Comment by captn3m0** [comment]
@adrianchifor It seems that the installation depends on your `pip` configuration, even for Python 3.6 (The above was on 3.7). I have the following in my `pip.conf` :

```
[install]
user = yes
no-binary = :all:
```

And that breaks the installation only for kapitan. Kapitan's source package on PyPi is broken because it doesn't ship with a requirements.txt file, and this ensures that the installation passes for all setups. (The right fix might be to just ensure that the package has `requirements.txt` file?)

**Comment by captn3m0** [comment]
Can be forced in Python 3.6 with:

```
pip install --no-binary kapitan kapitan
Collecting kapitan
  Downloading https://files.pythonhosted.org/packages/ca/df/6e4c4e9dc1c905aa9a3297276297565a5c5044c441012949235979913913/kapitan-0.23.0.tar.gz (91kB)
     |████████████████████████████████| 92kB 297kB/s 
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-a0y86qsz/kapitan/setup.py", line 55, in <module>
        pkgs, new_links = install_deps()
      File "/tmp/pip-install-a0y86qsz/kapitan/setup.py", line 41, in install_deps
        with open('requirements.txt', 'r') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-a0y86qsz/kapitan/
WARNING: You are using pip version 19.1, however version 19.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
# pip --version
pip 19.1 from /usr/local/lib/python3.6/site-packages/pip (python 3.6)
# python --version
Python 3.6.8
```

**Comment by uberspot** [comment]
The proper fix for this would actually be to include the `requirements.txt` file in pip.
@captn3m0 could you add the line `include requirements.txt` in `MANIFEST.in` and add it to this PR too? 
That controls what's packaged in PyPi. We can/should test that instead for a future-proof fix.

**Comment by adrianchifor** [comment]
requirements.txt has been included in package in https://github.com/deepmind/kapitan/pull/287, closing this.

**Pull request closed without merging**